src/components/CourseInfo/index.tsx

```diff
@@ -16,13 +19,65 @@ export default function CourseInfo({
 }: CourseInfoProps): React.ReactElement {
   const [{ oscar }] = useContext(ScheduleContext);
   const course = useMemo(() => oscar.findCourse(courseId), [oscar, courseId]);
+
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  useEffect(() => {
+    if (course) {
+      course
+        .fetchGpa()
+        .then(() => {
+          setIsLoaded(true);
+        })
+        .catch((err) => {
+          softError(
+            new ErrorWithFields({
+              message: 'error fetching course GPA',
+              source: err,
+              fields: {
+                courseId,
+                term: course.term,
+              },
+            })
+          );
+        });
+    }
+  }, [course, courseId]);
+
   const prerequisite = useMemo(() => {
     return course?.prereqs
-      ? serializePrereqs(course.prereqs as Exclude<CrawlerPrerequisites, []>)
+      ? serializePrereqs(
+          course.prereqs as Exclude<CrawlerPrerequisites, []>
+        ).replace(/^\(|\)$/g, '')
       : 'None';
   }, [course?.prereqs]);
   const credits = useMemo(() => course?.sections?.[0]?.credits, [course]);
 
+  /**
+   * Get the currterm + 5 terms prior to current term in course crit response
+   *
+   * Note that course critique as of Spr2025 only has data up to Spr2024
+   * This can make it incomplete (ie. From Spring 2025 to Spring 2024)
+   */
+  const offered_terms = useMemo(() => {
```

Comment:
```suggestion
  const offeredTerms = useMemo(() => {
```

Slight nitpick just to keep code style consistent. not that deep if u dont change it lol

if u do be sure to change it everywhere its used

Reply:
Good catch I will change 😃